chaste_codegen/_command_line_script.py

```diff
@@ -5,98 +5,161 @@
 import os
 from collections import OrderedDict
 
-import cellmlmanip
-
 import chaste_codegen as cg
 from chaste_codegen._script_utils import write_file
+from chaste_codegen.model_with_conversions import load_model_with_conversions
 
 
 # Link names to classes for converting code
-TRANSLATORS = OrderedDict([('Chaste', cg.NormalChasteModel),
-                           ('ChasteOpt', cg.OptChasteModel),
-                           ('CVODE', cg.CvodeChasteModel),
-                           ('CVODEWithDataClamp', cg.CvodeWithDataClampModel),
-                           ('BackwardsEuler', cg.BackwardEulerModel),
-                           ('RushLarsen', cg.RushLarsenModel),
-                           ('RushLarsenOpt', cg.RushLarsenOptModel),
-                           ('GeneralisedRushLarsen1', cg.GeneralisedRushLarsenFirstOrderModel),
-                           ('GeneralisedRushLarsen1Opt', cg.GeneralisedRushLarsenFirstOrderModelOpt),
-                           ('GeneralisedRushLarsen2', cg.GeneralisedRushLarsenSecondOrderModel),
-                           ('GeneralisedRushLarsen2Opt', cg.GeneralisedRushLarsenSecondOrderModelOpt)])
+TRANSLATORS = OrderedDict(
+    [('normal', (cg.NormalChasteModel, 'FromCellML', '', True)),
+     ('cvode', (cg.CvodeChasteModel, 'FromCellMLCvode', 'Cvode', True)),
+     ('cvode-data-clamp', (cg.CvodeWithDataClampModel, 'FromCellMLCvodeDataClamp', 'CvodeDataClamp', True)),
+     ('backward-euler', (cg.BackwardEulerModel, 'FromCellMLBackwardEuler', 'BackwardEuler', False)),
+     ('rush-larsen', (cg.RushLarsenModel, 'FromCellMLRushLarsen', 'RushLarsen', False)),
+     ('grl1', (cg.GeneralisedRushLarsenFirstOrderModel, 'FromCellMLGRL1', 'GRL1', False)),
+     ('grl2', (cg.GeneralisedRushLarsenSecondOrderModel, 'FromCellMLGRL2', 'GRL2', False))])
 
-TRANSLATORS_WITH_MODIFIERS = ('Chaste', 'ChasteOpt', 'CVODE', 'CVODEWithDataClamp')
+TRANSLATORS_OPT = OrderedDict(
+    [('normal', (cg.OptChasteModel, 'FromCellMLOpt', 'Opt', True)),
+     ('cvode', (cg.OptCvodeChasteModel, 'FromCellMLCvodeOpt', 'CvodeOpt', True)),
+     ('rush-larsen', (cg.RushLarsenOptModel, 'FromCellMLRushLarsen', 'RushLarsen', False)),
+     ('grl1', (cg.GeneralisedRushLarsenFirstOrderModelOpt, 'FromCellMLGRL1', 'GRL1', False)),
+     ('grl2', (cg.GeneralisedRushLarsenSecondOrderModelOpt, 'FromCellMLGRL2', 'GRL2', False))])

```

Comment:
Should these have 'Opt' appended to the class name?

Reply:
Would seem to make sense although pycml didn't seem to do this for those models I'll try with opt and see if I can still pass chaste tests